packages/@sanity/cli/templates/app-sanity-ui/src/App.css

```diff
@@ -0,0 +1,4 @@
+body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Would it make sense to use `createGlobalStyle` for this to avoid having multiple CSS methodologies in this template? Maybe even tossing it into `SanityUI.tsx`? Not blocking, just a thought!

Reply:
It may be! I have zero experience with Styled Components so if that's a more Styled Components-y thing to do let's do it.

Reply:
I filed a suggestion with the change, you'd just need to remove this CSS file (and the import) and test to make sure the global style I added works as expected :)

Reply:
Oh I didn't scroll down :|